File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/cheltuieli-arb-si-reprezentant-arb-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class CheltuieliArbSiReprezentantArbReportFilterModel {

	@JsonProperty("titular", String)
	public titular: string = null;
    
	@JsonProperty("organismId", Number)
    public organismId: number = null;
    
	@JsonProperty("comitet", String)
	public comitet: string = null;
    
	@JsonProperty("dataDecontDeLa", JsonDateConverter)
	public dataDecontDeLa: Date = null;

	@JsonProperty("dataDecontPanaLa", JsonDateConverter)
	public dataDecontPanaLa: Date = null;
	
	@JsonProperty("numarDecizie", String)
	public numarDecizie: string = null;

	@JsonProperty("valuta", String)
	public valuta: string = null;
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/cheltuieliArbSiReprezentantArb/CheltuieliArbSiReprezentantArbReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report.cheltuieliArbSiReprezentantArb;

import java.util.Date;


public class CheltuieliArbSiReprezentantArbReportFilterModel {

	private String titular;
	private Long organismId;
	private String comitet;
	private Date dataDecontDeLa;
	private Date dataDecontPanaLa;
	private String numarDecizie;
	private String valuta;

	public String getTitular() {
		return titular;
	}

	public void setTitular(String titular) {
		this.titular = titular;
	}

	public Long getOrganismId() {
		return organismId;
	}

	public void setOrganismId(Long organismId) {
		this.organismId = organismId;
	}

	public String getComitet() {
		return comitet;
	}

	public void setComitet(String comitet) {
		this.comitet = comitet;
	}

	public Date getDataDecontDeLa() {
		return dataDecontDeLa;
	}

	public void setDataDecontDeLa(Date dataDecontDeLa) {
		this.dataDecontDeLa = dataDecontDeLa;
	}

	public Date getDataDecontPanaLa() {
		return dataDecontPanaLa;
	}

	public void setDataDecontPanaLa(Date dataDecontPanaLa) {
		this.dataDecontPanaLa = dataDecontPanaLa;
	}

	public String getNumarDecizie() {
		return numarDecizie;
	}

	public void setNumarDecizie(String numarDecizie) {
		this.numarDecizie = numarDecizie;
	}

	public String getValuta() {
		return valuta;
	}

	public void setValuta(String valuta) {
		this.valuta = valuta;
	}

}
